src/components/common/InputField.jsx

```diff
@@ -17,10 +18,11 @@ const InputField = (props) => {
     checkConfirm,
     isButtonClicked,
     autoComplete,
+    Icon,
```

Comment:
**Warning**

Line 21:5:  'Icon' is assigned a value but never used   no-unused-vars